staging/src/k8s.io/apiserver/pkg/storage/cacher/cache_watcher.go

```diff
@@ -503,6 +503,20 @@ func (c *cacheWatcher) processInterval(ctx context.Context, cacheInterval *watch
 		klog.V(2).Infof("processing %d initEvents of %s (%s) took %v", initEventCount, c.groupResource, c.identifier, processingTime)
 	}
 
+	// send bookmark right now for watchlist request
+	if cacheInterval.bookmarkNewFunc != nil {
+		bookmarkEvent := &watchCacheEvent{
+			Type:            watch.Bookmark,
+			Object:          cacheInterval.bookmarkNewFunc(),
+			ResourceVersion: resourceVersion,
+		}
```

Comment:
We're missing the `InitialEventsAnnotationKey` here

Reply:
It doesn't matter. Because InitialEventsAnnotationKey annotation will be added in `convertToWatchEvent` method